scijava/scijava-javadoc-parser/src/main/java/org/scijava/javadoc/parser/PackageFilter.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2015 David Nault and contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.scijava.javadoc.parser;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.lang.model.element.Element;
+import javax.lang.model.element.ElementKind;
+import javax.lang.model.element.QualifiedNameable;
+
+class PackageFilter {
+
```

Comment:
Should give this file proper attribution - we didn't write it, it came directly from therapi

Reply:
Gone now!